sklearn/neighbors/_classification.py

```diff
@@ -37,6 +37,7 @@ class KNeighborsClassifier(KNeighborsMixin, ClassifierMixin, NeighborsBase):
 
     Read more in the :ref:`User Guide <classification>`.
 
+
```

Comment:
```suggestion
```

By removing this line, it should solve the CI failure.

Reply:
Thanks for pointing out the CI failure, I hadn't noticed it.